iina/ShooterSubtitle.swift

```diff
@@ -148,7 +140,3 @@ class Shooter {
     }
   }
 }
-
-extension Logger.Sub {
-  static let shooter = Logger.makeSubsystem("sub.shooter")
-}
```

Comment:
This was added in anticipation of log messages being added by someone who knows this provider.

There are problems with not being able to tell what is going wrong when trying to find and download subtitles. Some refactoring was done to share logging and reporting in `OnlineSubtitle`. However as can be seen in `OpenSubSubtitle` some logging should be done by the specific provider class.

Reply:
@low-batt Fixed: 6b202ba4703d6a8885379596bb79c0a96e4fa2e2